src/navability/graphql/Variable.jl

```diff
@@ -119,3 +119,36 @@ GQL_GETVARIABLESFILTERED = """
       }
     }
   }"""
+
+GQL_INITVARIABLE = """
```

Comment:
This should work. Just wanted to mention it could be simpler if you want. This is part of the thinking around always having just the two input objects. It will enable us to start building structs and types around the inputs more easily. I change to the gql API is just a change to a struct rather than in the gql layer, the calling function signature, and the code in between.

```julia
GQL_INIT_VARIABLE = """
  mutation test_init_variable(\$variable: InitVariableInput!, \$options: EmptyInputOptions) {
    test: initVariable(variable: \$variable, options:\$options) {
      context {
        eventId
      }
      status {
        state
        progress
      }
    }
  }
"""
```

Reply:
okay thanks -- I'm following the docs in api and still looks like the older (non-input-type) approach on p1?

i'm struggling with a Diana parsing error and trying to find it in CloudWatch Insights to see if I can figure why its breaking.

Reply:
![Screenshot from 2022-08-07 11-37-59](https://user-images.githubusercontent.com/6412556/183306007-60870eb0-7b10-4cbc-a9a8-1e7fc731894a.png)

Reply:
Let me ask the other way round.  The GQL you suggest is a cleaner way to do what I'm trying on current p1?

Reply:
So I'd make `struct InitVariableInput` in Julialand?

Reply:
![Screenshot from 2022-08-07 11-48-56](https://user-images.githubusercontent.com/6412556/183306493-71483cf4-10c7-45a2-843e-c19f7f223e23.png)

So `where` is the first field, but we cannot use that as a Julia keyword?

Reply:
That was the theory yes but alas restricted words kill us. For now, I guess you just have a make a dict with a key "where", but let's make an issue in SchemaCore to eliminate the use of where and document why.

Reply:
PS, also moved bandwidth down in visionary SchemaCore:
- https://github.com/NavAbility/SchemaCore/blob/5cf5c7b0b31ab9a42aff05cb63ac3335b0093d70/schemas/visionary/common/distribution.graphql#L36